packages/react-native/src/primitives/PhoneNumberField/PhoneNumberField.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { View } from 'react-native';
+import { Picker } from '@react-native-picker/picker';
+
+import { TextField } from '../TextField';
+import { PhoneNumberFieldProps } from './types';
+import { styles } from './styles';
+
+export default function PhoneNumberField({
+  defaultDialCode,
+  dialCodes,
+  disabled,
+  inputStyle,
+  onDialCodeChange,
+  pickerItemStyle,
+  pickerStyle,
+  style,
+  ...rest
+}: PhoneNumberFieldProps): JSX.Element {
+  const [selectedDialCode, setSelectedDialCode] = useState(defaultDialCode);
```

Comment:
Just curious: if `defaultDialCode` is `undefined`, then does the `Picker` default to showing the first `dialCode`?

Reply:
yes the Picker shows the first element in the list. Added a test for it!